spec/robots/dor_repo/was_crawl_dissemination/cdxj_generator_spec.rb

```diff
@@ -8,9 +8,16 @@
 
     let(:druid) { 'druid:dd116zh0343' }
     let(:sys_cmd) do
-      'TMPDIR=/tmp /opt/app/was/.local/bin/poetry ' \
-        'run cdxj-indexer /web-archiving-stacks/data/collections/xx123xx1234/dd/116/zh/0343/number1.warc ' \
```

Comment:
I found it more readable to break up the system command into meaningful components and then join them.